OpenOversight/tests/routes/test_api_v1.py

```diff
@@ -0,0 +1,77 @@
+import json
+from http import HTTPStatus
+
+import pytest
+from flask import current_app, url_for
+from sqlalchemy.orm import contains_eager, joinedload
+
+from OpenOversight.app.models.database import Assignment, Incident, Officer, Salary
+from OpenOversight.app.utils.constants import ENCODING_UTF_8
+
+
+@pytest.mark.parametrize("department_id", [1, 3000])
+def test_get_dept_attributes(client, session, department_id: int):
+    with current_app.test_request_context():
+        # Get officers
+        expected_officers = Officer.query.filter_by(department_id=department_id).count()
+
+        resp_officers = client.get(
+            url_for("v1.get_dept_officers", department_id=department_id),
+            follow_redirects=True,
+        )
+        officers = json.loads(resp_officers.data.decode(ENCODING_UTF_8))
+
+        assert resp_officers.status_code == HTTPStatus.OK
+        assert len(officers) == expected_officers
+
+        # Get assignments
+        expected_assignments = (
+            session.query(Assignment)
+            .join(Assignment.base_officer)
+            .filter(Officer.department_id == department_id)
+            .options(contains_eager(Assignment.base_officer))
+            .options(joinedload(Assignment.unit))
+            .options(joinedload(Assignment.job))
+            .all()
+        )
+
+        resp_assignments = client.get(
+            url_for("v1.get_dept_assignments", department_id=department_id),
+            follow_redirects=True,
+        )
```

Comment:
I could probably get rid of `follow_redirects`, but I think it might be worth keeping in case we modify the endpoint at some point.